src/CADL.Extension/Emitter.Csharp/package.json

```diff
@@ -34,7 +34,27 @@
     "files": [
         "dist/src/**"
     ],
+    "dependencies": {
+        "@autorest/csharp": "3.0.0-beta.20230309.1",
+        "fs-extra": "^10.1.0",
+        "js-yaml": "^4.1.0",
+        "json-serialize-refs": "^0.1.0-0",
```

Comment:
This package is not very widely used (we are the only dependents according to npmjs), and is a pre-release version. We try to avoid such dependencies in our GA packages.

Tracking: https://github.com/Azure/autorest.csharp/issues/3459